internals/models.py

```diff
@@ -349,7 +347,7 @@ def del_none(d):
   """
   Delete dict keys with None values, and empty lists, recursively.
   """
-  for key, value in d.items():
+  for key, value in list(d.items()):
```

Comment:
I thought that we could use items() in python3.  E.g., from the python3 tutorial:
https://docs.python.org/3/tutorial/datastructures.html#looping-techniques
And, when I try it in python3 interactively, items() seems to work.

Reply:
I think you need the list() only when the list is being stored in a variable, not when it is used only for iteration in a for-loop.

Reply:
Just remembered that it is caused by a different issue

  File "/usr/local/google/home/kyleju/chromium-dashboard/internals/models.py", line 350, in del_none
    for key, value in d.items():
RuntimeError: dictionary changed size during iteration

the d[key]` caused a self-modifying loop. We could use list here or iterate through the keys of the dict

Reply:
Ah, OK.